src/javascripts/components/tabs.js

```diff
@@ -43,32 +43,42 @@ AppTabs.prototype.activateAndToggle = function (event) {
   event.preventDefault()
   var $currentToggler = event.target
   var $currentTogglerSiblings = this.$module.querySelectorAll('[href="' + $currentToggler.hash + '"]')
-  var $tabContainer = this.$module.querySelector($currentToggler.hash)
+  var $tabContainer
+
+  try {
+    $tabContainer = this.$module.querySelector($currentToggler.hash)
+  } catch (exception) {
+    if ((typeof console !== 'undefined' || typeof console.log !== 'undefined')) {
```

Comment:
Does this swallow the exception?

Potentially this would make it less likely for us to be able to catch bugs since it would not break the build?

If we think it's useful to have a more helpful error message and that is true, we could consider throwing a custom error here rather than just logging it to the console?

Reply:
A good catch, thanks @nickcolley